src/api.v2/model/CellLevelMetaToExperiment.js

```diff
@@ -11,6 +11,15 @@ class CellLevelMetaToExperiment extends BasicModel {
   constructor(sql = sqlClient.get()) {
     super(sql, tableNames.CELL_LEVEL_TO_EXPERIMENT_MAP, fields);
   }
+
+  async setNewFile(experimentId, cellMetadataFileId) {
+    // Remove references to previous file
+    await this.delete({ experiment_id: experimentId });
```

Comment:
if there are multiple files associated with an experiment, will they be deleted too?

Reply:
There can't be multiple files associated with an experiment

Reply:
Good, then this code should be fine. How do we enforce the contraint that there can't be multiple files associated with an experiment?

Reply:
It is not enforced, but it cant happen, since adding a new entry, tries to delete references to previous ones before adding a new one.